src/internal.c

```diff
@@ -16813,7 +16813,7 @@ int DecodePrivateKey(WOLFSSL *ssl, word16* length)
     int CheckVersion(WOLFSSL *ssl, ProtocolVersion pv)
     {
 #ifdef WOLFSSL_TLS13
-    #ifndef WOLFSSL_TLS13_FINAL
+    #ifdef WOLFSSL_TLS13_DRAFT
```

Comment:
Why did you switch FINAL to DRADT and leave a TODO?

Reply:
Fixed. Removed all TODOs about version including in DRAFT18 specific code.